src/client/game.cpp

```diff
@@ -837,7 +836,6 @@ class Game {
 	Camera *camera = nullptr;
 	Clouds *clouds = nullptr;	                  // Free using ->Drop()
 	Sky *sky = nullptr;                         // Free using ->Drop()
-	Inventory *local_inventory = nullptr;
```

Comment:
Are you (kind reviewer) OK with this change?
 * No out-of-sync inventories
 * No memory duplication

On the cost of getting the inventory through `LocalPlayer` each time.

Reply:
Getting the inventory from `LocalPlayer` seems to just be a matter of doing `player->inventory`. That's perfectly fine IMO, unless there's a disadvantage that I don't notice. Even if there was a semi-negligible disadvantage, they're heavily outweighed by the advantage of no out-of-sync inventories.

We could make the pointer private, and add a const getter though, but that's for a different discussion. :)